platformio.ini

```diff
@@ -805,3 +806,13 @@ lib_deps =
   ${esp32.lib_deps}
   TFT_eSPI @ ^2.3.70
 board_build.partitions = ${esp32.default_partitions}
+
+# ------------------------------------------------------------------------------
+# Arduino® Nano ESP32
+# ------------------------------------------------------------------------------
+[env:arduino_nano_esp32]
+platform = https://github.com/platformio/platform-espressif32.git
+board = arduino_nano_esp32
+upload_protocol = esptool
+board_build.partitions = tools/WLED_ESP32_16MB.csv
+board_build.flash_mode = dio
```

Comment:
For proper builds, please add at least the following parts from [[env:esp32s3dev_8MB_PSRAM_opi]](https://github.com/Aircoookie/WLED/blob/4245767357dc609fa1f567b22a0817ddc3e3971e/platformio.ini#L507)
* `extends`
* `build_flags`
* `lib_deps`

Maybe you also need `build_unflags`, if some standard  USB flags are in conflict to defaults for your board.

Reply:
Done